index.html

```diff
@@ -127,7 +128,259 @@ <h2>Multi-Post Stories</h2>
+                    <ul>
+                        <li class="projectHTML">
+                            <ul>
+                                <li>html</li>
+                            </ul>
+                        </li>
+                        <li class="projectBootstrap">
+                            <ul>
+                                <li>bootstrap</li>
+                            </ul>
+                        </li>
+                        <li class="projectRuby">
+                            <ul>
+                                <li>ruby</li>
+                            </ul>
+                        </li>
+                    </ul>
+                </div>
+                <div class="projectButton">
+                    <button type="button">See Project</button>
+                </div>
+            </div>
+
+            <div class="project">
+                <div class="projectHeader">
+                    <h2>Profesional Art Printing Data</h2>
+                </div>
+
+                <div class="projectBody">
+                    <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+                    <ul>
+                        <li class="projectHTML">
+                            <ul>
+                                <li>html</li>
+                            </ul>
+                        </li>
+                        <li class="projectBootstrap">
+                            <ul>
+                                <li>bootstrap</li>
+                            </ul>
+                        </li>
+                        <li class="projectRuby">
+                            <ul>
+                                <li>ruby</li>
+                            </ul>
+                        </li>
+                    </ul>
+                </div>
+                <div class="projectButton">
+                    <button type="button">See Project</button>
+                </div>
+            </div>
+        </section>
+
+        <section class="whoIsNemwel">
+            <div class="Nemweldesc">
+                <div class="nemweldescleft">
+                    <div class="nemwelHeader">
+                        <h2>About me</h2>
+                    </div>
+                    <div class="nemwelbody">
+                        <p>Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+                        <div class="nemwelbtn">
+                            <button type="button">Get My Resume</button>
+                        </div>
+                    </div>
+                </div>
+
+                <div class="nemweldescright">
+                    <img src="images/aboutmebgimage.png" alt="aboutme background image">
```

Comment:
- [ ] Is a project requirement that decorations are always background images, not `img` tags. I see you're a using an `img` tag here please let's fix this and add this image as a background image.

Reply:
Thank @mike2611 you for this review
- I have made the necessary changes.